### DISCUSSION TITLE
关于开往维护组春节放假的通知和新年祝福

### DISCUSSION THREAD

**Discussion by linlinzzo** [travellings-link/travellings#1408] [公告]
开往将于1月21日至1月26日放假，收录申请的审核将顺延至2月26日（根据维护组内部讨论，设立一个月的申请考察期）。在放假期间，开往维护组成员的响应时间无法保证，如遇不可用性报告，可以考虑暂时使用其他博客聚合项目：
十年之约虫洞 https://www.foreverblog.cn/go.html
中文博客列表导航-随机跳转到生活类博客 https://zhblogs.ohyee.cc/go?tags=%E7%94%9F%E6%B4%BB

2022年是开往项目极具挑战、充满争议的一年，感谢各位陪开往走过2022。自2022年4月项目优化审核以来，开往的成员越来越多。开往倡导开放的网络，这也正是项目名字的灵感来源。我们希望通过开往这个平台，各位站长可以一起交朋友，新博客主可以得到更好的帮助。2023年新年即将到来，开往项目愿与各位站长一起踏上新的Travelling。

2023年三月，开往将迎来他的三岁生日。让我们预祝开往项目生日快乐！😃

林林
2023年1月20日

**Comment by PrintlnPan** [comment]
![AD343DB14A09D97D30FCBE255FBB768A](https://user-images.githubusercontent.com/117614950/213842475-b349d0e8-73d0-4ce2-a44d-43b73a9beb6c.jpg)
